Gemfile.lock

```diff
@@ -149,7 +149,7 @@ GEM
       aws-sigv4 (~> 1.4)
     aws-sigv4 (1.5.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    bcrypt (3.1.17)
+    bcrypt (3.1.18)
```

Comment:
Some performance improvements, dropping GVL during hashing (Ruby's Global VM Lock for MRI) https://my.diffend.io/gems/bcrypt/3.1.17/3.1.18